src/doc/rustdoc/src/scraped-examples.md

```diff
@@ -0,0 +1,55 @@
+# Scraped examples
```

Comment:
Can you move the part about how this feature is unstable up to the top of the page? Otherwise, the first few paragraphs make it look like this feature is enabled-by-default.